.github/workflows/test-appsync-utils.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 env:
   LOCALSTACK_AUTH_TOKEN: ${{ secrets.LOCALSTACK_API_KEY }}
-  TEST_IMAGE_NAME: public.ecr.aws/lambda/nodejs:16
+  TEST_IMAGE_NAME: public.ecr.aws/lambda/nodejs:18
```

Comment:
Er... we are still using 16 (for better or worse...) https://github.com/localstack/localstack-ext/blob/1f952f304e39d1d066a4d703a2727b395bf8af9e/localstack-pro-core/localstack/pro/core/services/appsync/mapping.py#L25

However we _do_ want to move our JS execution to the node in the container image, so this is a good move, thanks! 👍